templates/site/js/node.js

```diff
@@ -113,6 +115,7 @@ document.addEventListener("DOMContentLoaded", function () {
             node_menu_item.innerText = json[key].name;
             if (index < initial_node_count) {
                 const h3_node_menu_item = document.createElement("a");
+                h3_node_menu_item.id = key;
```

Comment:
The `onclick` handler you introduce above is on the `node_menu_item` not on the `h3_node_menu_item`. This means that when the initial option loads it will click this `<a>` tag, not the `<div>` that actually has the onclick handler. 

You should set the id of the `node_menu_item` to this `id` instead so that the element with the `id` matches the element that has the onclick handler.

Reply:
Set the id on node_menu_item.